Podcast.xcodeproj/project.pbxproj

```diff
@@ -15,6 +15,7 @@
 		0C8122551FAA32DA00A414AE /* SearchITunesViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0C8122541FAA32DA00A414AE /* SearchITunesViewController.swift */; };
 		0CB18BBA204346BE0074B525 /* NSAttributedString+.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0CB18BB9204346BE0074B525 /* NSAttributedString+.swift */; };
 		0CBD32D01F8F0F2B007115E7 /* DispatchTime+.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0CBD32CF1F8F0F2B007115E7 /* DispatchTime+.swift */; };
+		2BE87E1A20476F6700240B31 /* NullProfileCollectionViewCell.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2BE87E1920476F6700240B31 /* NullProfileCollectionViewCell.swift */; };
```

Comment:
With iPhone X the tableview insets are not being correctly readjusted ...

iphone 8:
<img width="269" alt="screen shot 2018-03-02 at 4 34 57 pm" src="https://user-images.githubusercontent.com/14966713/36922984-c4d4a0a4-1e37-11e8-8c53-96686bcc948f.png">

iphone X:
<img width="279" alt="screen shot 2018-03-02 at 4 35 04 pm" src="https://user-images.githubusercontent.com/14966713/36922988-c72ce42e-1e37-11e8-8ec2-6510b882b171.png">

This is probably is `ViewController` which is where we adjust tableview inset accordingly. 

Another example: 
<img width="275" alt="screen shot 2018-03-02 at 4 33 45 pm" src="https://user-images.githubusercontent.com/14966713/36923010-dfc6575e-1e37-11e8-8956-1501fc7ca18b.png">